File: GO/Par_ou_Impar.go
package main

import "fmt"

func main(){
	var x int = 0
	fmt.Print("\nDigite o número desejado: ")
	fmt.Scanf("%d", &x);
	
	if x%2 == 0{		 
    		fmt.Println("\nO Número digitado é par\n");
    	}else{
    		fmt.Println("\nO Número digitado é ímpar\n");
    	}		
}

File: C/Par_ou_Impar.c
#include <stdio.h>

int main(){
 	int x;

 	printf("\nDigite o Número desejado:\n");
  	scanf("%d",&x);
  	//Compara se o número digitado é par
   	if (x%2 == 0){		 
    		printf("\n O Número digitado é par\n");
    	}
    	//Compara se o número digitado é ímpar
   	else if (x%2 != 0){
    		printf("\n O Número digitado é ímpar\n");
    	}
    	return 0;
} 
